corehq/apps/cloudcare/templates/formplayer/case_detail.html

```diff
@@ -9,9 +9,9 @@
 <script type="text/template" id="detail-view-item-template">
   <th><%- header %></th>
   <% if (style.displayFormat === 'Graph') { %>
-  <td><iframe srcdoc="<%= data %>" height="300" width="300"></iframe></td>
+  <td><iframe srcdoc="<%- data %>" height="300" width="300"></iframe></td>
```

Comment:
This seems like one of the riskier changes. Are case list graphs specifically in the QA plan?

Reply:
Thanks for pointing it out. I was not able to reproduce this on local. My understanding for this change was that `data` variable would be a URL but I feel that I think I am wrong here.
Can you guide me on how can I put a graph in case details list?

Reply:
I'm pretty sure it's the HTML that renders the graph, including javascript. [These docs](https://confluence.dimagi.com/display/RD/Graphing+in+HQ) walk through graphing setup, which is quite a few steps. [This app](https://staging.commcarehq.org/a/bosco/apps/view/0168bb256b4e45b396de87c2af4ae4ae) on staging has a graph set up, you're welcome to modify/copy it. I think the simplest test would be to add special characters to some piece of text that shows on the graph, like an axis label (`x-title` / `y-title`).